sdk/storage/azure-storage-blob/azure/storage/blob/_download.py

```diff
@@ -40,23 +55,25 @@ def process_range_and_offset(start_range, end_range, length, encryption_options,
     return (start_range, end_range), (start_offset, end_offset)
 
 
-def process_content(data, start_offset, end_offset, encryption):
+def process_content(data: Any, start_offset: int, end_offset: int, encryption: Dict[str, Any]):
     if data is None:
         raise ValueError("Response cannot be None.")
 
-    content = b"".join(list(data))
+    content = b"".join(list(data))  #type: ignore
```

Comment:
This (and line 58) is because I don't know how to type-hint data properly. Line 62 will complain because `list()` is looking for a mutable-sequence but the type of data when checking with debugger is:
<img width="508" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/101599632/983f3383-963c-4e7d-ba8f-65ab09325e85">
😭

Reply:
Use `Any`